Support Java 9

Plugin uses gradle 4.2, which fails on computers with Java 9.0.1 only installed.

How to reproduce:
1. Clear gradle caches
2. Clone repository
3. Execute gradlew.bat build

Error: 

```
C:\git\gradle-msbuild-plugin [master ≡]> .\gradlew.bat build
Downloading https://services.gradle.org/distributions/gradle-4.2-bin.zip
...................................................................
Unzipping C:\g\wrapper\dists\gradle-4.2-bin\cl9xe1l8k8ophj72k8rv6xbu7\gradle-4.2-bin.zip to C:\g\wrapper\dists\gradle-4.2-bin\cl9xe1l8k8ophj72k8rv6xbu7

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine java version from '9.0.1'.
```

--- a/gradle/wrapper/gradle-wrapper.properties
+++ b/gradle/wrapper/gradle-wrapper.properties
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.9-all.zip

